reaktive/src/nativeCommonTest/kotlin/com/badoo/reaktive/looperthread/LooperThreadTest.kt

```diff
@@ -0,0 +1,85 @@
+package com.badoo.reaktive.looperthread
+
+import com.badoo.reaktive.utils.Lock
+import com.badoo.reaktive.utils.atomic.AtomicBoolean
+import com.badoo.reaktive.utils.synchronized
+import platform.posix.usleep
+import kotlin.system.getTimeMillis
+import kotlin.system.getTimeNanos
+import kotlin.test.Test
+import kotlin.test.assertFalse
+import kotlin.test.assertTrue
+
+class LooperThreadTest {
+
+    @Test
+    fun executes_task() {
+        val isExecuted = AtomicBoolean()
+        val lock = Lock()
+        val condition = lock.newCondition()
+        val thread = LooperThread()
+        val startTime = getTimeMillis() + 200L
+
+        thread.schedule(Unit, startTime) {
+            lock.synchronized {
+                isExecuted.value = true
+                condition.signal()
+            }
+        }
+
+        lock.synchronized {
+            while (!isExecuted.value) {
+                condition.await()
```

Comment:
Add timeout to `condition.await()`, otherwise our test will be stuck on CI for 30 minutes (AFAIK).